fit/core-reference/src/main/resources/log.properties

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I was not able to access to `core-test.properties`from `TestFileAuditAppender.java`